app/assets/stylesheets/common/base/user.scss

```diff
@@ -651,6 +651,79 @@
     align-items: center;
     gap: 1em;
   }
+
+  .pref-auth-tokens {
```

Comment:
This section was moved here from `base/discourse.scss` (in part because it also shares some styles with the passkeys list).